### PR TITLE
Fetch Exercise Cards - Fetch Search Form - Exercise Cards Pagination

### PR DESCRIPTION
**Pull request by tufankaracam**

### CODE REVIEW

File: src/js/create-pagination.js [hunk 1]
```diff
@@ -1,17 +1,106 @@
...
+        ${page + 2}
+      </button>
+    `;
+  }
+
+  if (page + 3 <= totalPages) {
+    buttons += `
+        <button class="button" data-page="1" disabled>
+          ...
+        </button>
```
Comment (pavlomatchak) [inline src/js/create-pagination.js:73]: Do we have so many pages?

File: src/js/main.js [hunk 2]
```diff
@@ -1,20 +1,88 @@
...
+  });
+
+});
+
+content.addEventListener('click', async e => {
+  const item = e.target.closest('.category-wrap');
+
+  if (!item) return;
+  searchForm.classList.remove('is-hide');
+  muscles = item.getAttribute('name');
```
Comment (pavlomatchak) [inline src/js/main.js:57]: Muscles is a category name. It's not a good idea to call variable this

File: src/js/create-pagination.js (hunk 1 above)
Comment (tufankaracam) [reply src/js/create-pagination.js:73]: I saw up to 17 pages. Especially on mobile it ruins design.

File: src/js/main.js (hunk 2 above)
Comment (pavlomatchak) [reply src/js/main.js:54]: Got it. Didn't know that there are so many

File: src/js/create-pagination.js (hunk 1 above)
Comment (pavlomatchak) [reply src/js/create-pagination.js:73]: Got it. Didn't know that there are so many

File: src/js/main.js (hunk 2 above)
Comment (tufankaracam) [reply src/js/main.js:57]: It is parameter of exercises endpoint. If you have any recommendation i can change it.

Comment (pavlomatchak) [reply src/js/main.js:57]: If it's exercise, call it exercise :) Muscles is too specific besides it doesn't represent what this variable contains

**Review by pavlomatchak — APPROVED** [review]

**Pull request merged**